<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>691</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dx_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dy_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dz_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="move_btn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="dx_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dx:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dz_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dy_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dy:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_btn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>320</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="az_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>290</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ay_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="az_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>αz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ay_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>αy:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ax_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rot_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QLabel" name="ax_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>αx:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ky_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ky:</string>
    </property>
   </widget>
   <widget class="QLabel" name="kx_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>kx:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kx_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="kz_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>kz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scale_lb">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_btn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабировать</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kz_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ky_sb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="load_le">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла модели</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_btn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
